plugins/module_utils/_data_set_utils.py

```diff
@@ -273,6 +274,85 @@ def _execute_command(command):
     module = AnsibleModuleHelper(argument_spec={})
     return module.run_command(command)
 
+def _submit_jcl(jcl_uss_path, job_name):
+    executions =[]
+    command = "jsub -f '{0}'".format(jcl_uss_path)
+
+    rc, stdout,stderr = _execute_command(command)
+    executions.append(
+    _execution(
+        name="Submit jcl job {0} for {1}".format(jcl_uss_path, job_name),
```

Comment:
Does the file path name make sense here in the first format?
Would read like `Submit jcl job /tmp/file/path for DFHRMUTL`
Also we should capitalize JCL in any externals